src/kOS.Safe/Compilation/KS/Compiler.cs

```diff
@@ -1703,6 +1706,20 @@ private void VisitToggleStatement(ParseNode node)
             AddOpcode(new OpcodeStore());
         }
 
+		private void VisitHudtxtStatement(ParseNode node)
+		{
+			NodeStartHousekeeping(node);
+			{
+				VisitNode(node.Nodes[2]);
+				VisitNode(node.Nodes[4]);
+				VisitNode(node.Nodes[6]);
+				VisitNode(node.Nodes[8]);
+				VisitNode(node.Nodes[10]);
+				VisitNode(node.Nodes[12]);
+				AddOpcode(new OpcodeCall("hudtxt()"));
+			}
+		}
+
```

Comment:
This can probably be re-indented by an auto-reformat tool on our end, but I really hate using them because if we run all files through them we preclude the abiliy to use intelligent indents in places where it makes sense to look a bit different.  (i.e. places where people have padded spaces into the code to make columns of stuff line up, for example.)